parquet/metadata/column_chunk.go

```diff
@@ -349,12 +357,12 @@ type EncodingStats struct {
 // Finish finalizes the metadata with the given offsets,
 // flushes any compression that needs to be done, and performs
 // any encryption if an encryptor is provided.
-func (c *ColumnChunkMetaDataBuilder) Finish(info ChunkMetaInfo, hasDict, dictFallback bool, encStats EncodingStats, metaEncryptor encryption.Encryptor) error {
```

Comment:
I think `encryptor` isn't used in this function anymore, remove the argument?

Reply:
I was trying to avoid breaking changes. But I guess i can remove it entirely